spec/duration.html

```diff
@@ -1254,6 +1257,8 @@ <h1>RoundDuration ( _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _sec
           1. Else,
             1. Assert: _relativeTo_ has an [[InitializedTemporalDate]] internal slot.
           1. Let _calendar_ be _relativeTo_.[[Calendar]].
+        1. Else,
+          1. Let _calendar_ be *undefined*.
```

Comment:
This is somewhat subtle - and maybe there should be a note, but this is probably fine. _calendar_ is only used if _unit_ is "year", "month", or "week", and in that case, we've verified _relativeTo_ is not undefined in step 3.

Reply:
Ahh, yeah that is subtle. Nice catch. So no change needed here either?

I'm happy to add an editorial note here, but not sure what the format is and where to add it. Is there an example in another AO that I can copy?  If not, want to make a suggestion with format and location and I can fill in the actual text if needed?

Reply:
Maybe just
```suggestion
        1. Else,
          1. NOTE: _calendar_ will not be used below.
```

Reply:
Sounds good. I'm going to include this change in a separate PR and will close this PR.